core/dbt/artifacts/schemas/manifest/v12/manifest.py

```diff
@@ -0,0 +1,180 @@
+    )
+    macros: Mapping[UniqueID, Macro] = field(
+        metadata=dict(description=("The macros defined in the dbt project and its dependencies"))
+    )
+    docs: Mapping[UniqueID, Documentation] = field(
+        metadata=dict(description=("The docs defined in the dbt project and its dependencies"))
+    )
+    exposures: Mapping[UniqueID, Exposure] = field(
+        metadata=dict(
+            description=("The exposures defined in the dbt project and its dependencies")
+        )
+    )
+    metrics: Mapping[UniqueID, Metric] = field(
+        metadata=dict(description=("The metrics defined in the dbt project and its dependencies"))
+    )
+    groups: Mapping[UniqueID, Group] = field(
+        metadata=dict(description=("The groups defined in the dbt project"))
+    )
+    selectors: Mapping[UniqueID, Any] = field(
+        metadata=dict(description=("The selectors defined in selectors.yml"))
+    )
+    disabled: Optional[Mapping[UniqueID, List[GraphMemberNode]]] = field(
+        metadata=dict(description="A mapping of the disabled nodes in the target")
+    )
+    parent_map: Optional[NodeEdgeMap] = field(
+        metadata=dict(
+            description="A mapping from child nodes to their dependencies",
+        )
+    )
+    child_map: Optional[NodeEdgeMap] = field(
+        metadata=dict(
+            description="A mapping from parent nodes to their dependents",
+        )
+    )
+    group_map: Optional[NodeEdgeMap] = field(
+        metadata=dict(
+            description="A mapping from group names to their nodes",
+        )
+    )
+    saved_queries: Mapping[UniqueID, SavedQuery] = field(
+        metadata=dict(description=("The saved queries defined in the dbt project"))
+    )
+    semantic_models: Mapping[UniqueID, SemanticModel] = field(
+        metadata=dict(description=("The semantic models defined in the dbt project"))
+    )
+    metadata: ManifestMetadata = field(
+        metadata=dict(
+            description="Metadata about the manifest",
+        )
+    )
+    unit_tests: Mapping[UniqueID, UnitTestDefinition] = field(
+        metadata=dict(
+            description="The unit tests defined in the project",
+        )
+    )
+
+    @classmethod
+    def compatible_previous_versions(cls) -> Iterable[Tuple[str, int]]:
+        return [
+            ("manifest", 4),
+            ("manifest", 5),
+            ("manifest", 6),
+            ("manifest", 7),
+            ("manifest", 8),
+            ("manifest", 9),
+            ("manifest", 10),
+            ("manifest", 11),
+        ]
+
+    @classmethod
+    def upgrade_schema_version(cls, data):
+        """This overrides the "upgrade_schema_version" call in VersionedSchema (via
+        ArtifactMixin) to modify the dictionary passed in from earlier versions of the manifest."""
+        manifest_schema_version = get_artifact_schema_version(data)
+        if manifest_schema_version <= 10:
```

Comment:
It should be! Let me make sure I haven't clobbered anything during a merge...

Reply:
our latest released version is still v11: https://schemas.getdbt.com/

This also looks consistent with whats on main: https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/contracts/graph/manifest.py#L1730-L1732

I'm hesitant to bump this here because it will likely need additional testing & implementation. I'll open up an issue we can take on as part of the dbt/artifacts refactor

Reply:
Issue: https://github.com/dbt-labs/dbt-core/issues/9438